src/auth/src/errors.rs

```diff
@@ -63,7 +63,7 @@ impl CredentialError {
     /// # Arguments
     /// * `is_retryable` - A boolean indicating whether the error is retryable.
     /// * `message` - The underlying error that caused the auth failure.
-    pub(crate) fn from_str<T: Into<String>>(is_retryable: bool, message: T) -> Self {
+    pub fn from_str<T: Into<String>>(is_retryable: bool, message: T) -> Self {
```

Comment:
comment: As I suspected, this was a necessary change to use `CredentialError` outside of the `google-cloud-auth` crate. Thanks.